spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -227,80 +254,6 @@
     end
   end
 
-  describe "#assign_visibility?" do
```

Comment:
Why'd you remove the `#assign_visibility?` block of tests? It doesn't look like you touched this method's implementation.